sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3892,13 +3899,13 @@ pi_result piEventRelease(pi_event Event) {
     ZE_CALL(zeEventDestroy(Event->ZeEvent));
 
     auto Context = Event->Context;
-    ZE_CALL(Context->decrementAliveEventsInPool(Event->ZeEventPool));
+    Context->decrementAliveEventsInPool(Event->ZeEventPool);
```

Comment:
Same question as for Context->getFreeSlotInExistingOrNewPool() above.

Reply:
Added error check